xDrip/Scenes/Settings/BaseSettings/BaseSettingsViewController.swift

```diff
@@ -50,21 +64,36 @@ class BaseSettingsViewController: UIViewController {
         self.viewModel = viewModel
         if animated {
             // TODO: Implement
+            tableView.reloadSections(IndexSet(integersIn: 0 ..< viewModel.sections.count), with: .automatic)
         } else {
             tableView.reloadData()
         }
     }
     
     private func handleSelectionForNormalCell(_ cell: BaseSettings.Cell) {
         switch cell {
-        case .disclosure(_, _, let selectionHandler) :
+        case .disclosure(_, _, let selectionHandler):
             selectionHandler()
         default: break
         }
     }
     
     private func handleSingleSelection(indexPath: IndexPath, callback: (Int) -> Void) {
-        // TODO: Implement
+        guard let section = viewModel?.sections[indexPath.section] else { return }
+        
+        switch section {
+        case let .singleSelection(cells, _, header, footer, selectionHandler):
+            viewModel?.sections[indexPath.section] = .singleSelection(cells: cells,
```

Comment:
Fix code style - place a newline after opening `(` and put each argument on a new line (currently `header` and `footer` are on the same line).